pyrft/cluster_inference.py

```diff
@@ -234,22 +234,22 @@ def cluster_tdp_brain(imgs, design, contrast_matrix, mask, n_bootstraps = 100, f
     mask = mask > 0
 
     # For each cluster calculate the TDP
-    for L in np.arange(n_contrasts):
+    for l in np.arange(n_contrasts):
         # Get the clusters of the test-statistic
-        cluster_im, cluster_sizes = pr.find_clusters(pvalues_3d[..., L], cdt, below = 1, mask = mask, min_cluster_size = min_cluster_size)
+        cluster_im, cluster_sizes = pr.find_clusters(pvalues_3d[..., l], cdt, below = 1, mask = mask, min_cluster_size = min_cluster_size)

```

Comment:
line size should not exceed 80 (pep8...)

Reply:
Okay I've added a \ in order to break up the lines, is that what you had in mind?

Reply:
It's still too long

Reply:
But each line is below 80 now right i.e:      
 cluster_im, cluster_sizes = pr.find_clusters(pvalues_3d[..., l], cdt, \
                   below = 1, mask = mask, min_cluster_size = min_cluster_size)

It showed up as a problem in my linter before but now it doesn't. (I hadn't figured out how to use the \ to fix it by going across multiple lines, which is why I hadn't previously fixed it.)